ansible/README.md

```diff
@@ -1,10 +1,42 @@
 # Ansible Playbooks
 
+## Database Initialization
+
+The following databases are created by the `postgresql/init.yml` playbook:
+
+| Database      | Owner   |
+| ------------- | ------- |
+| de            | de      |
+| notifications | de      |
+| metadata      | de      |
+| de_releases   | de      |
+| grouper       | grouper |
+| qms           | de      |
+| unleash       | de      |
+| k3s           | de      |
+
+The owner users are configurable through the `dbms_connection_user` and `grouper_connection_user` group_vars.
+
+Migrations are run for the `de`, `metadata`, `notifications`, and `de_releases` databases. The `k3s` database is initialized by the installation process for the k3s cluster and the `grouper` database is handled by it's own playbook since it's fairly complicated. The `qms`
+database is created here, but populated by the `qms` service. `unleash` is not yet initialized by this playbook.
```

Comment:
QMS is currently configured not to apply migrations to the `qms` database by default, so it might be useful to have this playbook run the QMS migrations as well. These migrations are currently in the QMS repository, but I'd be open to moving them if that makes life easier.

Reply:
This is fixed.